src/index.test.ts

```diff
@@ -126,7 +193,7 @@ describe('get()', () => {
     const client = createClient<paths>();
     fetchMocker.mockResponseOnce(() => ({ status: 200, body: JSON.stringify(mockData) }));
     const { data, error, response } = await client.get('/post/{post_id}', {
-      params: { path: { post_id: 'my-post' } },
+      params: { path: { post_id: 'my-post' }, query: {} },
```

Comment:
In fixing one bug for query params, I introduced another one where if a path has _any_ possible query params, the whole object is required 🤔 

I’ll have to come back to this one.